entrypoint.sh

```diff
@@ -24,4 +24,8 @@ code-suggester $INPUT_COMMAND \
 --message="$INPUT_MESSAGE" \
 --force="$INPUT_FORCE" \
 --maintainers-can-modify="$INPUT_MAINTAINERS_CAN_MODIFY" \
---git-dir="$INPUT_GIT_DIR"
\ No newline at end of file
+--git-dir="$INPUT_GIT_DIR"; then
+    exit 0
+else
+    exit 1
```

Comment:
You could do "set -e" at the top of the bash script instead. It tells bash to fail if any command in the script fails.

Reply:
for script: `exit_code.sh`

```
set -e

code-suggester pr \
--upstream-repo="dne" \
--upstream-owner="TomKristie" \
--description="describe" \
--title="title" \
--branch="something" \
--primary="master" \
--message="some message" \
--git-dir="./kubernetes"
```
after running `./exit_code.sh`
and then `echo $?` it outputs exit code 0. According to the docs it looks like it needs to have an error code of 1 in order for the step to fail.

Does `set -e` make the script fail with error 1?